website/content/docs/glossary.mdx

```diff
@@ -0,0 +1,93 @@
+---
+layout: docs
+page_title: Glossary of Terms
+sidebar_title: Glossary
+description: |-
+  Vault Glossary.
+---
+
+# Glossary
+
+This page collects brief definitions of some of the technical terms used in the
+documentation for Vault.
+
+- [Audit Device](#audit-device)
+- [Auth Method](#auth-method)
+- [Barrier](#barrier)
+- [Client Token](#client-token)
+- [Secret](#secret)
+- [Secrets Engine](#secrets-engine)
+- [Server](#server)
+- [Storage Backend](#storage-backend)
+
+### Audit Device
+
+An audit device is responsible for managing audit logs.
+Every request to Vault and response from Vault goes through the configured
+audit devices. This provides a simple way to integrate Vault with multiple
+audit logging destinations of different types.
+
+### Auth Method
+
+An auth method is used to authenticate users or applications
+which are connecting to Vault. Once authenticated, the auth method returns the
+list of applicable policies which should be applied. Vault takes an
+authenticated user and returns a client token that can be used for future
+requests. As an example, the `userpass` auth method uses a username and
+password to authenticate the user. Alternatively, the `github` auth method
+allows users to authenticate via GitHub.
+
+### Barrier
+
+The barrier is cryptographic steel and concrete around the
```

Comment:
Are "steel" and "concrete" technical terms in cryptography? If not, I would recommend avoiding metaphorical language.

Reply:
Modified the entire definition based on Nick's feedback.